RDKShell/RDKShell.cpp

```diff
@@ -4758,8 +4758,22 @@ namespace WPEFramework {
                   keyClient = keyInputInfo.HasLabel("callsign")? keyInputInfo["callsign"].String(): "";
                 }
                 lockRdkShellMutex();
-                ret = CompositorController::generateKey(keyClient, keyCode, flags, virtualKey);
-                gRdkShellMutex.unlock();
+                std::vector<std::string> clientList;
+                 CompositorController::getClients(clientList);
+                bool targetFound = false;
+                for (size_t i=0; i<clientList.size(); i++)
+               {
+                if (strcasecmp(clientList[i].c_str(),keyClient.c_str()) == 0)
+                {
+                    targetFound = true;
+                    break;
+                }
+                }
```

Comment:
Let's use `std::find(clientList.begin(), clientList.end(), keyClient) != clientList.end()` instead.  Please ensure the indent spacing count matches the style of the method as well.